language/move-prover/src/boogie_helpers.rs

```diff
@@ -182,7 +216,7 @@ pub fn boogie_declare_global(env: &GlobalEnv, name: &str, ty: &Type) -> String {
     format!(
         "var {} : Value where {};",
```

Comment:
I recall there was some confusion earlier about the meaning of "where" clause in Boogie.  I have never used it myself.  Do we have clarity about what "where" does?

Reply:
From the "this is boogie" doc, it maintains an invariant (which must be a stateless expression) over the lifetime of a global variable, so does exactly what we want here.